src/kafka-util/src/client.rs

```diff
@@ -625,16 +623,105 @@ pub fn get_partitions<C: ClientContext>(
     Ok(partition_ids)
 }
 
+/// Default to true as they hey have no downsides <https://github.com/confluentinc/librdkafka/issues/283>.
+pub const DEFAULT_KEEPALIVE: bool = true;
+/// The `rdkafka` default.
+/// - <https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md>
```

Comment:
idk, maybe this is just inane + pedantic but trying to duplicate default values into MZ rather than just not specifying the setting feels bad to me. Are there other places you saw where we do this? If we intentionally do this elsewhere, maybe change here and let's bring the others into this format? Or maybe I'm just being overly cautious today and this is NBD.

Reply:
i went back and forth on this and decided on this route to match what we do elsewhere (like with pg and ssh) where we actively configure the defaults when we create connections, rather than leaving them `None`. It IS slightly different here because rdkafka is the one controlling the defaults, not us, but I have a slight preference to doing it consistently; also, like I mention below, `transaction.timeout.ms` and `socket.timeout.ms` have a weird relationship and setting one and not the other is going to force us to validate against the rdkafka default anyways :(